html5validator/cli.py

```diff
@@ -90,8 +93,8 @@ def main():
     args, extra_args = parser.parse_known_args()
 
     if args.config is not None:
-        args = parse_yaml(args.config, args)
-        LOGGER.debug(args)
+        args, extra_args = parse_yaml(args.config, args)
```

Comment:
I didn't realize this before, but overwriting all command line arguments is really not great. A config file should serve as a default that can be overridden by command line args. A todo item for later.

Reply:
The issue with this is that most command-line arguments have a default so we would need to have a list of what the default arguments are to compare to what was passed.